CVLByExamples/ReadOnlyReentrancy/README.md

```diff
@@ -0,0 +1,15 @@
+This directory contains two versions of a bank contract and a spec that applies the readOnlyReentrancy 
+builtin rule to them.
+
+In VulnerableBank.sol the rule readOnlyReentrancy fails because of a possible Read-Only Reentrancy weakness at external call site at file contracts/VulnerableBank.sol, in function withdraw().
```

Comment:
every file name should be wrapped in backticks, e.g. "`VulnerableBank...."

Reply:
done